ironfish-cli/src/commands/chain/download.ts

```diff
@@ -215,23 +215,28 @@ export default class Download extends IronfishCommand {
       }
     }
 
-    // use a standard name, 'snapshot', for the unzipped database
-    const snapshotDatabasePath = this.sdk.fileSystem.join(this.sdk.config.tempDir, 'snapshot')
-    await this.sdk.fileSystem.mkdir(snapshotDatabasePath, { recursive: true })
-
-    CliUx.ux.action.start(`Unzipping ${snapshotPath}`)
-    await this.unzip(snapshotPath, snapshotDatabasePath)
-    CliUx.ux.action.stop('done')
-
-    const chainDatabasePath = this.sdk.fileSystem.resolve(this.sdk.config.chainDatabasePath)
-
-    CliUx.ux.action.start(
-      `Moving snapshot from ${snapshotDatabasePath} to ${chainDatabasePath}`,
-    )
-    // chainDatabasePath must be empty before renaming snapshot
-    await fsAsync.rm(chainDatabasePath, { recursive: true, force: true, maxRetries: 10 })
-    await fsAsync.rename(snapshotDatabasePath, chainDatabasePath)
-    CliUx.ux.action.stop('done')
+    try {
+      const chainDatabasePath = this.sdk.fileSystem.resolve(this.sdk.config.chainDatabasePath)
+
+      // chainDatabasePath must be empty before unzipping snapshot
+      CliUx.ux.action.start(
+        `Removing existing chain data at ${chainDatabasePath} before importing snapshot`,
+      )
+      await fsAsync.rm(chainDatabasePath, { recursive: true, force: true, maxRetries: 10 })
+      CliUx.ux.action.stop('done')
+
+      // ensure that chainDatabasePath exists
+      await fsAsync.mkdir(chainDatabasePath, { recursive: true })
+
+      CliUx.ux.action.start(`Unzipping ${snapshotPath} to ${chainDatabasePath}`)
+      await this.unzip(snapshotPath, chainDatabasePath)
+      CliUx.ux.action.stop('done')
+    } catch (e) {
+      this.logger.error(
+        `Failed to import snapshot. Run \`ironfish chain:download --path ${snapshotPath}\` to retry import without download`,
```

Comment:
I'm usually pretty wary about adding things like this, because they can confuse users a lot but if you think it's useful I'm ok putting it in. For instance, could we remove this error and just do this automatically for the user? If we see a snapshot at the location just ask if they want to import that instead then remove this?

Reply:
I see what you mean.

I think I'll remove the message here, but I think it could be tricky to automatically import a snapshot if they have multiple snapshot files lying around